yarn-project/end-to-end/src/shared/cli.ts

```diff
@@ -8,23 +8,34 @@ const TRANSFER_BALANCE = 3000;
 
 export const cliTestSuite = (
   name: string,
-  setup: () => Promise<PXE>,
+  setup: () => Promise<{
+    /**
+     * The PXE instance.
+     */
+    pxe: PXE;
+    /**
+     * The URL of the PXE RPC server.
+     */
+    rpcURL: string;
+  }>,
   cleanup: () => Promise<void>,
   debug: DebugLogger,
-  rpcUrl = 'http://localhost:8080',
 ) =>
   describe(name, () => {
     let cli: ReturnType<typeof getProgram>;
     let pxe: PXE;
     let existingAccounts: CompleteAddress[];
     let contractAddress: AztecAddress;
     let log: (msg: string) => void;
+    let rpcURL = '';
 
     // All logs emitted by the cli will be collected here, and reset between tests
     const logs: string[] = [];
 
     beforeAll(async () => {
-      pxe = await setup();
+      const { pxe: pxeService, rpcURL: pxeURL } = await setup();
+      pxe = pxeService;
+      rpcURL = pxeURL;
```

Comment:
```suggestion
      ({ pxe, rpcURL } = await setup());
```
I _think_ this syntax works

Reply:
Thanks